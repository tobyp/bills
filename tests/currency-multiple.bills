#!./bills

%set XTS 2.0  # XTS is the test currency code
%set XTT 4.0

A	B.	4.00

A	B.	2.00*XTS

A	B.	1.00*XTT

A	B.	2.00*XTS

A	B.	4.00
