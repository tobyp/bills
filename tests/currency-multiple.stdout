todo: B pays 20.0 to A
